pkg/providers/amifamily/al2023.go

```diff
@@ -39,7 +39,7 @@ func (a AL2023) DescribeImageQuery(ctx context.Context, ssmProvider ssm.Provider
 	ids := map[string]Variant{}
 	for arch, variants := range map[string][]Variant{
 		"x86_64": {VariantStandard, VariantNvidia, VariantNeuron},
-		"arm64":  {VariantStandard},
+		"arm64":  {VariantStandard, VariantNvidia},
```

Comment:
Do we have E2E validation for launching these combination of instances? The value might make sense

Reply:
No, but we don't have E2E tests all of the supported AMI variants generally. I'm not sure there's sufficient value in adding one vs the functional tests. It would really just serve as a canary for AMI availability.